src/validatePrTitle.js

```diff
@@ -30,13 +30,17 @@ module.exports = async function validatePrTitle(
   }
   const result = parser(prTitle, parserOpts);
 
+  function removeCapitalLetter(text) {
+    return text.charAt(0).toLowerCase() + text.substring(1);
+  }
+
   function printAvailableTypes() {
     return `Available types:\n${types
       .map((type) => {
         let bullet = ` - ${type}`;
 
         if (types === defaultTypes) {
-          bullet += `: ${conventionalCommitTypes.types[type].description}`;
+          bullet += `: ${removeCapitalLetter(conventionalCommitTypes.types[type].description)}`;
```

Comment:
Instead of outputting:
```
Available types:
 - feat: A new feature
 - fix: A bug fix
 - docs: Documentation only changes
 - style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
 - refactor: A code change that neither fixes a bug nor adds a feature
 - perf: A code change that improves performance
 - test: Adding missing tests or correcting existing tests
 - build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
 - ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
 - chore: Other changes that don't modify src or test files
 - revert: Reverts a previous commit
 ```
 
 
This will output:
```
Available types:
 - feat: a new feature
 - fix: a bug fix
 - docs: documentation only changes
 - style: changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
 - refactor: a code change that neither fixes a bug nor adds a feature
 - perf: a code change that improves performance
 - test: adding missing tests or correcting existing tests
 - build: changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
 - ci: changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
 - chore: other changes that don't modify src or test files
 - revert: reverts a previous commit
 ```